packages/alfa-rules/src/sia-er62/rule.ts

```diff
@@ -186,8 +161,14 @@ export default Rule.Atomic.of<Page, Element>({
               );
               const distinguishableContrast = Set.from(
                 Array.from(nonLinkElements)
```

Comment:
We probably don't need to pile up `Set.from(Array.from(…))`. `Set.from()` should work on any `Iterable` and `nonLinkElements` is already a `Sequence` (iirc), so we can just use `Set.from`.